### PR TITLE
feat: restructure templates [fixes DXJ-526]

### PR DESCRIPTION
**Pull request by shamsartem**

### CODE REVIEW

File: src/lib/init.ts [hunk 1]
```diff
@@ -312,92 +295,228 @@ type InitTSorJSProjectArg = {
...
+        (() => {
+          throw new Error(\`Button with id="\${fnName}" is not found\`);
+        })();
+
+      buttonEl.addEventListener("click", () => {
+        runAquaFunction(fn);
+      });
+    });
+  });
+})();
```
Comment (akim-bow) [inline src/lib/init.ts:430]: No need to use `requestAnimationFrame`  optimization here i think. You make very simple operation and just once. That function usually used for DOM animation which optimized to be updated only when your monitor showing next frame.

File: src/lib/init.ts [hunk 2]
```diff
@@ -312,92 +295,228 @@ type InitTSorJSProjectArg = {
...
+    })(),
+  ]);
+}
+
+function getIndexHTMLContent(isJS: boolean) {
+  return `<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="data:,">
```
Comment (akim-bow) [inline src/lib/init.ts:346]: didn't understand this. Maybe we can add fluence icon or something here?

Comment (shamsartem) [reply src/lib/init.ts:346]: we would have to add icon code to template as well, we probably can do it. Chrome always requests favicon.ico even if you remove this and you get 404, this is the simplest way to make chrome shut up

Comment (shamsartem) [reply src/lib/init.ts:346]: inserted base64 encoded image, seems fine

File: src/lib/init.ts (hunk 1 above)
Comment (shamsartem) [reply src/lib/init.ts:430]: restructured this part with createElement like in other places in the code so there is no need to find by id and stuff

**Pull request merged**